environmentlogger/environmental_logger_json2netcdf.py

```diff
@@ -90,6 +90,12 @@
                     u'us': {"original":"microsecond", "SI":"second", "power":1e-6},
                     u'ppm': {"original":"pascal meter-2", "SI":"pascal meter-2", "power":1}, 
                     '': ''}
+
+_CF_STANDARDS    = {u'precipitation' : "liquid_water_equivalent_precipitation_rate",
```

Comment:
We use the name `precipitation_flux`, see 
https://pecan.gitbooks.io/pecan-documentation/content/developers_guide/Adding-an-Input-Converter.html